flink-python/pyflink/common/types.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, *args, **kwargs):
             self._from_dict = True
         else:
             self._values = list(args)
+            self._from_dict = False
```

Comment:
nit: can we do this outside the if with self._from_dict = kwargs;

Reply:
This would set `self._from_dict` to `{'a': 'pyflink', 'b': True}` rather than a boolean which feels a little unintuitive